doc/compiler/il/MainlineAndOutOfLineCode.md

```diff
@@ -0,0 +1,47 @@
+<!--
+Copyright (c) 2021, 2021 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Mainline and Out of Line Code
```

Comment:
Try to use line breaks when writing .md documents. GitHub renders around ~150 characters so that seems like a reasonable number. Personally I use 120.

Using line break makes reviewing easier as the smallest granularity you can review on GitHub is one line. So placing paragraphs on the same line makes it more difficult to review parts of the PR. Additionally `git blame` works on a line basis so it becomes messy when making future changes here.

Reply:
I'm mistaken, it looks like it is ~93 characters on the PR view, but ~150 on the review tab of the PR.

Reply:
updated in 6738cf6